packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -1,31 +1,121 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
+import QuickLinks from './components/quick-links/QuickLinks';
+import NavItems from './components/nav-items/NavItems';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
- /*
- * Content to be displayed as the name
- */
-  name: PropTypes.string,
+  test: PropTypes.string,
+  help: PropTypes.string,
+  isMobileNavOpen: PropTypes.bool, // this is proof of concept. should we keep this?
+  quickLinks: PropTypes.arrayOf(PropTypes.object),
+  navItems: PropTypes.arrayOf(PropTypes.object),
+  mainContent: PropTypes.node,
 };
 
 const defaultProps = {
-  name: 'default',
+  isMobileNavOpen: false,
+  test: 'hello world',
+  help: 'help',
+  mainContent: null,
+  quickLinks: [
```

Comment:
default props should have this as empty... if you need test examples.. create them in the tests directory for nightwatch and you can use them to develop as well.